Bounce/dist/presence.js

```diff
@@ -117,8 +117,34 @@ presence.on("UpdateData", () => {
     }
      else if(document.location.pathname.startsWith("/portal")) {
       presenceData.state = "Browsing Staff Panel";
-      presenceData.details = "Logging in"
-  }
+      presenceData.details = "Logging in" 
```

Comment:
maybe add logging in as a default in the switch?

Reply:
default is not required.